app/src/components/landing/pricing/PricingIndexPage/PricingPageFooter/index.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from "react";
+import { Row, Col, Typography, Space } from "antd";
+import { EVENTS, trackRequestDocumentClicked } from "./analytics";
+import RequestDocsModal from "./RequestDocsModal";
+import { getFunctions, httpsCallable } from "firebase/functions";
+
+const { Title, Text, Link } = Typography;
+
+const styles = {
+  footer: {
+    padding: "40px 20px",
+  },
+  container: {
+    maxWidth: "70vw",
+    margin: "0 auto",
+  },
+  title: {},
+  companyInfo: {
+    marginTop: "20px",
+    textAlign: "center",
+    color: "var(--neutrals-gray-300)",
+  },
+};
+
+const PricingPageFooter = () => {
+  const [isRequestDocsModalOpen, setisRequestDocsModalOpen] = useState(false);
+
+  const handleDocRequiredOnClick = () => {
+    trackRequestDocumentClicked();
```

Comment:
should we also track which docRequired was clicked, inside trackRequestDocumentClicked as event property

Reply:
It won't be required for now since we'd be reaching out via email anyway